### PR TITLE
Ilya lashkin hw3

### PR DESCRIPTION
**Pull request by lalalash**
Hi! Here is main part of HW3. Waiting for comments.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 *.DS_Store
+.vscode
```
Comment (katemalash) [inline .gitignore:2]: good idea.

File: lashkin/hw3/t1.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+var2 = input ('Provide 2nd variable! ')
+
+print(f'Your 1st variable is {var1}, your 2nd variable is {var2}')
+print('And now swap is coming...')
+
+sleep(2.5)
+
+var1, var2 = var2, var1
+
+print(f'So, your 1st variable now is {var1}, your 2nd variable is {var2}')
```
Comment (katemalash) [inline lashkin/hw3/t1.py:14]: This's the simplest way to do this task. You've done it good. sleep was not necessary, but you've used it and it's good

File: lashkin/hw3/t2.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#take input
+_bytes = input ('How many bytes do you want to convert? ')
```
Comment (katemalash) [inline lashkin/hw3/t2.py:2]: why have you used "_" to name the variable?

File: lashkin/hw3/t2.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+#check if input is propper
+if _bytes.isnumeric() == False:
```
Comment (katemalash) [inline lashkin/hw3/t2.py:5]: Very good point to check if the entered number is applicable. Nice

File: lashkin/hw3/t2.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    print('You provided NOT a number/negative number! Try again using positive numbers.')
+    exit()
+
+_bytes = float(_bytes)
+
+#calculating
+kilobytes = _bytes / 1000
```
Comment (katemalash) [inline lashkin/hw3/t2.py:12]: Actually, 1 kilobyte has 1024 bytes, not 1000

File: lashkin/hw3/t2.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+megabytes = _bytes / 1000000
```
Comment (katemalash) [inline lashkin/hw3/t2.py:13]: The same as previous

File: lashkin/hw3/t3.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
+#take input
+hight = input ('Hight: ')
+width = input ('Width: ')
+
+#check if input is propper
+if hight.isnumeric() == False or width.isnumeric() == False:
```
Comment (katemalash) [inline lashkin/hw3/t3.py:6]: good point. You also can check each value separately.

File: lashkin/hw3/t3.py [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+hight = float(hight)
+width = float(width)
+
+if hight == 0 or width == 0:
+    print('You can\'t provide 0 lenghts! Try again using positive number.')
+    exit()
+
+#calculate
+area = width * hight
+perimetr = 2 * (width + hight)
```
Comment (katemalash) [inline lashkin/hw3/t3.py:19]: Operator within the braces has higher priority, so it shouldn't has spaces around

File: lashkin/hw3/t3.py [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+print(f'Area is {area}, perimetr is {perimetr}.')
+
+if width == hight:
+    print('Nice fact: your rectangle is square :)')
```
Comment (katemalash) [inline lashkin/hw3/t3.py:24]: In general task is done good

File: lashkin/hw3/t4.py [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    exit()
+
+radius = float(radius)
+
+if radius == 0:
+    print('You can\'t provide 0 lenghts! Try again using positive number.')
+    exit()
+
+#calculate
+area = pi * radius ** 2
```
Comment (katemalash) [inline lashkin/hw3/t4.py:19]: Exponential operator has higher priority, so, PEP8

File: lashkin/hw3/t4.py [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+perimetr = 2 * pi * radius
+
+area = round(area, 3)
+perimetr = round(perimetr, 3)
+
+print(f'Area is {area}, perimetr is {perimetr}.')
```
Comment (katemalash) [inline lashkin/hw3/t4.py:25]: Task is done good in general

File: lashkin/hw3/t5.py [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+sideA = float(sideA)
+sideB = float(sideB)
+
+if sideA == 0 or sideB == 0:
+    print('You can\'t provide 0 lenghts! Try again using positive number.')
+    exit()
+
+#calculate
+
+hypotenusSqrt = sideA ** 2 + sideB ** 2
```
Comment (katemalash) [inline lashkin/hw3/t5.py:22]: PEP8

File: lashkin/hw3/t5.py [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+hypotenus = math.sqrt(hypotenusSqrt)
+hypotenus = round(hypotenus, 2)
+
+print(f'Hypotenus is {hypotenus}.')
+
+if sideB == sideA:
+    print('Nice fact: your triangle is isosceles :)')
```
Comment (katemalash) [inline lashkin/hw3/t5.py:29]: Task is done good in general

**Review by katemalash — APPROVED** [review]
Home work is done good. Now you can fix some minor issues and merge it.

File: lashkin/hw3/t2.py (hunk 3 above)
Comment (lalalash) [reply lashkin/hw3/t2.py:2]: Just thought name "byte" is reserved in Python, but it's not.

Comment (katemalash) [reply lashkin/hw3/t2.py:2]: Ok, it's not good practice to start name with "_". Internal variables names start with "_", but it's incapsulation principle, we will  talk about them the next year.

**Pull request merged**